traffic_prophet/countmatch/tests/test_growthfactor.py

```diff
@@ -1,108 +1,165 @@
+import pytest
 import hypothesis as hyp
 import numpy as np
-import pandas as pd
 import statsmodels.api as sm
 
-from .. import reader
+from .. import permcount as pc
+from .. import derivedvals as dv
 from .. import growthfactor as gf
 
-from ...data import SAMPLE_ZIP
 
+def get_single_ptc(sample_counts, cfgcm_test, count_id):
```

Comment:
Same as with `derivedvals`, change the names of arguments to highlight they don't need to be fixtures.